### PR TITLE
Update pmemkv module and CI

### PR DESCRIPTION
**Pull request by lukaszstolarczuk**
I've added some fixes in pmemkv module, including usage of pmemkv-root from maven repository (instead of a custom pmemkv-java installation).

I've also updated the CI workflow to only run our module (the other ones make the build red all the time).

### CODE REVIEW

**Comment by igchor** [comment]
LGTM

File: pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java [hunk 1]
```diff
@@ -103,23 +103,23 @@ public ByteBuffer toByteBuffer(Map<String, ByteIterator> entries) {
...
-        if (engineName == null) {
-          engineName = "cmap";
-        }
+        // use cmap as default engine
+        String engineName = props.getProperty(ENGINE_PROPERTY, "cmap");
+
         String path = props.getProperty(PATH_PROPERTY);
         if (path == null) {
-          throw new DBException(PATH_PROPERTY + " is obligatory to run");
+          throw new DBException(PATH_PROPERTY + " is obligatory to run PmemKV module");
```
Comment (KFilipek) [inline pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java]: module -> client

File: pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java [hunk 2]
```diff
@@ -103,23 +103,23 @@ public ByteBuffer toByteBuffer(Map<String, ByteIterator> entries) {
...
         }
         String size = props.getProperty(SIZE_PROPERTY);
         if (size == null) {
-          throw new DBException(SIZE_PROPERTY + " is obligatory to run");
+          throw new DBException(SIZE_PROPERTY + " is obligatory to run PmemKV module");
```
Comment (KFilipek) [inline pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java]: module -> client

File: pmemkv/README.md [hunk 3]
```diff
@@ -16,46 +16,52 @@ LICENSE file.
...
 You need to clone the repository and compile PmemKV module.
 
-```
-git clone git://github.com/brianfrankcooper/YCSB.git
-cd YCSB
-mvn -pl site.ycsb:pmemkv-binding -am package
-```
+    git clone git://github.com/brianfrankcooper/YCSB.git
+    cd YCSB
+    mvn -pl site.ycsb:pmemkv-binding -am package
```
Comment (KFilipek) [inline pmemkv/README.md:43]: Why not code block?
We can also use indentation in code sections.

**Review by KFilipek — CHANGES_REQUESTED** [review]

File: pmemkv/README.md (hunk 3 above)
Comment (lukaszstolarczuk) [reply pmemkv/README.md:43]: I've used the same approach as in other modules. For some reason, they use this style (which produces the same output as our ```sh approach - see for yourself: https://github.com/lukaszstolarczuk/YCSB/tree/use-pmemkv-from-maven/pmemkv)

File: pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java (hunk 2 above)
Comment (lukaszstolarczuk) [reply pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java]: Done

File: pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java (hunk 1 above)
Comment (lukaszstolarczuk) [reply pmemkv/src/main/java/site/ycsb/db/pmemkv/PmemKVClient.java]: Done

File: pmemkv/README.md (hunk 3 above)
Comment (KFilipek) [reply pmemkv/README.md:43]: Okey, then LGTM

**Review by KFilipek — APPROVED** [review]
LGBT

**Pull request merged**